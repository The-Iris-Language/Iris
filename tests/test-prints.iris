.. Tests print, println, printerr, and printi, our built-in print functions
class Main () {
    public:
        int univ main() {
            string whoa = "jshagdf";
            Olympus.printerr("error");
            Olympus.print("no newline                  hi there");
            Olympus.println(whoa);
            Olympus.printi(12);
            Olympus.println("");
        }
}