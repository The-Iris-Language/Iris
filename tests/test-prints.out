no newline                  hi therejshagdf
12
